build_tools/travis/install.sh

```diff
@@ -47,3 +47,6 @@ pip install -r requirements.txt
 
 # Install punkt tokenizer
 python -m nltk.downloader punkt
+
+# Install spacy data
+python -m spacy.en.download all
```

Comment:
the tests complained that we didn't have this, but i'm not sure. I was frankly surprised as well, since it was a non-issue before but spacy recently had a new release so it could be related to that. It's a small download anyway (50 MB)...